### DISCUSSION TITLE
Disable plug-in by URL

### DISCUSSION THREAD

**Discussion by Bobuntuu** [Woundorf/foxreplace#357] [General]
Can we get the ability to disable Fox-replace for a list of URLs?
I have problems with js-related text in iframes, image and video rendering issues when FR is on; e.g., bbc.com/news loading with no images or video players.
If I could just disable FR for a few specific pages I'd be happy.

**Comment by Woundorf** [comment]
Yes, this is explained in the [FAQ](https://github.com/Woundorf/foxreplace/wiki/FAQ), linked from the `Help` item in the FoxReplace button.

[How can I exclude a website from being replaced?](https://github.com/Woundorf/foxreplace/wiki/FAQ#how-can-i-exclude-a-website-from-being-replaced)
Add a hyphen `-` at the beginning, e.g. `-example.com`. Then it will be displayed in red to show that it will be excluded from application.

**Reply by Bobuntuu** [reply]
Hi Woundorf, thanks for the reply.
While the exclusion (-bbc.com) works for text replacements, the issue I'm experiencing, that makes me want to exclude the domain in the first place, still exists. 
I use https://www.bbc.com/news/world as my primary news source and, while the initial page loads without issue (screenshot 1), almost every subsequent linked page loads without any images or videos rendered (screenshot 2), whether in the same tab or a new one, until I disable FoxReplace (screenshot 3).  Occasionally, if I reload the page ten times or so, it will load fully and all will be well on that page, but that is rare.  On re-loading the pages, the images appear for a flash before disappearing again. If I disable FoxReplace and re-load the page, it renders perfectly, but if I then disable the extension and reload the page again, the problem returns.
Perhaps I should also mention that, with the exclusion active, even the text replacements activate for less than a second, before reverting to the original text.
Unfortunately, by excluding this domain and/or disabling the extension, I'm forced to suffer the awful English that BBC reporters seem intent on using.
 
Please note that I have tried disabling all other browser extensions to see if there are any conflicts, but that had no effect.

![Screenshot from 2023-10-13 16-17-42](https://github.com/Woundorf/foxreplace/assets/54530557/8055e2d1-2da5-4ba3-8c16-fc10b75d659f)

![Screenshot from 2023-10-13 16-47-20](https://github.com/Woundorf/foxreplace/assets/54530557/81f0fe4b-9139-4922-8211-05d1f74ddfd3)

![Screenshot from 2023-10-13 16-57-19](https://github.com/Woundorf/foxreplace/assets/54530557/d5ad10b8-3d6e-4223-b0d7-f069916effc7)

**Reply by Woundorf** [reply]
If you only want to replace text, as I understand, do you have substitutions without HTML? And replace URLs and replace scripts both disabled?
